### PR TITLE
Created trainer.py

### PR DESCRIPTION
**Pull request by BrysonParker1796**
Not the most familiar with inheritance in Python yet, but I think I did this right? I wasn't sure about how the as_dict should work exactly. Can y'all take a look and let me know if this is right?

### CODE REVIEW

**Comment by santosderek** [comment]
**Question:** What is t_time and t_focus supposed to be? 

**Also** I just created a UML diagram (Database_UML in our shared folder): https://drive.google.com/drive/u/1/folders/1tf0l9pRkR4SbL1fkvkAaZYMNST_VVXCL

**Additional Functionality:** 

- Could you also add a variable called `trainees` of type `list`.
- Add function `add_trainee(user_id: str): None`
- Add function `remove_trainee(user_id: str): None`

**Review by santosderek — CHANGES_REQUESTED** [review]
- Trainer's `__init__` should call `super().__init__()` with all the parameters of User.
- as_dict should return the result of `super().as_dict()` as shown as `**super().as_dict()` within the newly created, and returned, dictionary. 

Below is an example: 

```
class a(): 
    def __init__(self, something):
        self.something = something
    
    def as_dict(self): 
        return {'something': self.something}


class b(a): 
    def __init__(self, something, other):
        super().__init__(something)
        self.other = other

    def as_dict(self):
        return { **super().as_dict(), 'other' : self.other }


test = b(something='a', other='b')
print (test.as_dict())
```

**Comment by BrysonParker1796** [comment]
Ahh ok, still getting a handle on inheritance in Python. Thanks for the feedback!

File: vitality/trainer.py [hunk 1]
```diff
@@ -1,21 +1,17 @@
+from user import *
```
Comment (santosderek) [inline vitality/trainer.py:1]: `from .user import User` not wildcard, and missing `.`

**Review by santosderek — CHANGES_REQUESTED** [review]

**Review by santosderek — APPROVED** [review]

**Pull request merged**